test/e2e/storage/testsuites/base.go

```diff
@@ -684,3 +695,17 @@ func skipVolTypePatterns(pattern testpatterns.TestPattern, driver TestDriver, sk
 		framework.Skipf("Driver supports dynamic provisioning, skipping %s pattern", pattern.VolType)
 	}
 }
+
+func tryFunc(f func()) error {
+	var err error
+	if f == nil {
+		return fmt.Errorf("nil func received")
```

Comment:
If you turn this into `return nil` (i.e. allow tryFunc to be called with `nil`) then you can simplify all call locations by removing the `if` check.

Reply:
I have removed `if` check

Reply:
That still treats passing of a nil func as an error. What I meant with "return nil" is this code:

```
func tryFunc(f func()) error {
    if f == nil {
        return nil
    }
    var err error
    ...
```